e2e/e2eutil/consul.go

```diff
@@ -212,7 +212,7 @@ func CreateConsulToken(t *testing.T, client *capi.Client, namespace, policyID st
 		Description: "An e2e test token",
 	}, opts)
 	require.NoError(t, err, "failed to create consul acl token")
-	return token.SecretID
+	return token.SecretID, token.AccessorID
```

Comment:
Aren't there a bunch of callers that need to be updated?

Reply:
Yep, but they're all in ENT: hashicorp/nomad-enterprise#1337

Reply:
Ah!